### DISCUSSION TITLE
Figure

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#221] [Component suggestions]
<img width="480" alt="visuele weergave van het figure component" src="https://github.com/nl-design-system/backlog/assets/545207/60d54c77-5f16-46ab-a893-32433570705d">

We willen graag meer leren over het Figure component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1708610473670309)

---

## Naam
Figure

## Doel
Zelfstandige inhoud, zoals een afbeelding, met mogelijk een verklarende tekst.

## Gerelateerde componenten
[Image](https://github.com/orgs/nl-design-system/discussions/330), [Video](https://github.com/orgs/nl-design-system/discussions/299)

## Onderzoek
[Figure element - MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure)
Documentatie van HTML element `<figure>` inclusief voorbeelden.

**Comment by jeffreylauwers** [comment]
## Den Haag

- [Storybook](https://nl-design-system.github.io/denhaag/?path=/docs/css-data-display-image--docs)
- [GitHub](https://github.com/nl-design-system/denhaag/tree/main/components/Image)
- [Figma - Huidig](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/%F0%9F%8C%BB-Den-Haag-Design-System?type=design&node-id=8553-29678&mode=design&t=Pzl7lAESxldoteNX-4)

---

### Screenshots / Varianten

Default
![Scherm­afbeelding 2024-02-22 om 15 55 56](https://github.com/nl-design-system/backlog/assets/545207/c6a0797f-6cf9-4a47-8c81-51066c490b96)

Image with caption text
![Scherm­afbeelding 2024-02-22 om 15 56 43](https://github.com/nl-design-system/backlog/assets/545207/e46f8a24-38f5-4cb4-9db9-6c8f4c9bda03)

Image with download link
![Scherm­afbeelding 2024-02-22 om 17 20 45](https://github.com/nl-design-system/backlog/assets/545207/4add9e10-b015-4637-987b-14c1e49c4d58)

Image only
![Scherm­afbeelding 2024-02-22 om 17 21 18](https://github.com/nl-design-system/backlog/assets/545207/4c2b48d5-4333-4ab3-a55b-8dea06099142)



---

### Notities
- Figure gedrag is terug te vinden bij Image component.
- In code binnen het figure element met figcaption voor de tekst.

**Comment by jeffreylauwers** [comment]
## Open Gemeenten

- [Tekst en media - Documentatie website](https://componenten.lenteveld.nl/tekst-en-media)
- [Alleen afbeeldingen - Documentatie website](https://componenten.lenteveld.nl/alleen-afbeeldingen)
- [Patternlab](https://patternlab.lenteveld.nl/index.html)

---

### Screenshots / Varianten

**Documentatie website**
Tekst en media
![Scherm­afbeelding 2024-02-26 om 11 12 08](https://github.com/nl-design-system/backlog/assets/545207/70309635-3d99-48ad-a455-b88022ac6457)

Alleen afbeeldingen
![Scherm­afbeelding 2024-02-26 om 11 37 21](https://github.com/nl-design-system/backlog/assets/545207/7b7c26d2-2ee3-42f1-a19b-6e7fb7649c53)

**Patternlab**

[Media Gallery Caption](https://patternlab.lenteveld.nl/patterns/01-molecules-002-Generic-Content-Elements-Partials-Media-Media-Gallery-Caption/01-molecules-002-Generic-Content-Elements-Partials-Media-Media-Gallery-Caption.html)
![Scherm­afbeelding 2024-02-26 om 11 37 52](https://github.com/nl-design-system/backlog/assets/545207/6a5de2ce-eba2-45a6-a48c-0aa337828bfb)

[Media Gallery LinkExternalCaption](https://patternlab.lenteveld.nl/patterns/01-molecules-002-Generic-Content-Elements-Partials-Media-Media-Gallery-LinkExternalCaption/01-molecules-002-Generic-Content-Elements-Partials-Media-Media-Gallery-LinkExternalCaption.html)
![Scherm­afbeelding 2024-02-26 om 11 38 19](https://github.com/nl-design-system/backlog/assets/545207/88ef12b4-634d-4b5c-a72c-93386f806cd3)

---


### Notities
- Onder de naam: Media Gallery
- In code binnen het figure element met figcaption voor de tekst.
- In Patternlab veel vormen beschikbaar, ook als interactief element.

**Comment by jeffreylauwers** [comment]
## Rotterdam

[Figma](https://www.figma.com/file/ZWSC4gCrOXRUR9UX3aoZ8x/Gemeente-Rotterdam-Design-System?type=design&node-id=576-28132&mode=design&t=1ldypRdo1oqP9n9r-4)

---

### Screenshots / Varianten

**Figma**
![Scherm­afbeelding 2024-02-26 om 11 54 26](https://github.com/nl-design-system/backlog/assets/545207/d4f99306-16fe-4b66-a7a4-ad36a4512714)

---

### Notities
- Onder de naam: Content Image

**Comment by jeffreylauwers** [comment]
## Utrecht

- [Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-figure--docs)
- [GitHub](https://github.com/nl-design-system/utrecht/tree/main/components/figure)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-02-26 om 12 05 35](https://github.com/nl-design-system/backlog/assets/545207/bc5d9d4e-1cb8-4225-9137-1add9e260ff8)

---

### Notities
- Figcaption over meerdere regels inclusief een link.

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Figure op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=figure)

**Comment by eslook** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Figure op het Candidate bord](https://github.com/orgs/nl-design-system/projects/80/views/1?filterQuery=Figure)
